src/NuGet.Core/NuGet.Commands/RestoreCommand/DependencyGraphResolver.cs

```diff
@@ -165,27 +166,24 @@ public async Task<ValueTuple<bool, List<RestoreTargetGraph>, RuntimeGraph>> Reso
 
                 LibraryRangeIndex[] rootedDependencyPath = new[] { rootProjectRefItem.LibraryRangeIndex };
 
-                _ = findLibraryEntryCache.GetOrAddAsync(
```

Comment:
I think splitting refactoring/renaming from the code would make it so much easier to review.